src/Mvc/Mvc.ViewFeatures/src/Infrastructure/UnsupportedJavaScriptRuntime.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.JSInterop;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures
+{
+    internal class UnsupportedJavaScriptRuntime : IJSRuntime
```

Comment:
As per comments on the other PR, suggest renaming this to `PrerenderingJavaScriptRuntime`, and changing the error messages to:

```
JavaScript interop calls cannot be issued during server-side prerendering, because the page has not yet loaded in the browser. Prerendered components must wrap any JavaScript interop calls in conditional logic to ensure those interop calls are not attempted during prerendering.
```

I know that's a bit lengthy, but people will definitely hit this because they are coding naively, so it's good if the message tells them what sort of code fix they should try.

Reply:
Done